salt/ufw/etc_ufw/sysctl.conf

```diff
@@ -32,10 +56,42 @@ net/ipv4/conf/default/log_martians=0
 #net/ipv4/tcp_fin_timeout=30
 #net/ipv4/tcp_keepalive_intvl=1800
 
+# Ignore Directed pings
+#net/ipv4/icmp_echo_ignore_all = 1
+
 # Uncomment this to turn off ipv6 autoconfiguration
 #net/ipv6/conf/default/autoconf=1
 #net/ipv6/conf/all/autoconf=1
 
 # Uncomment this to enable ipv6 privacy addressing
 #net/ipv6/conf/default/use_tempaddr=2
 #net/ipv6/conf/all/use_tempaddr=2
+
+# Performance Recommended Settings:
```

Comment:
The following settings are not related to security: they are more directed towards performance: 
https://www.cyberciti.biz/faq/linux-tcp-tuning/

Do you think it's a good idea to separate these settings into another sysctl-sls related file?